demo/src/test/java/com/workday/autoparse/json/demo/JsonParserTest.java

```diff
@@ -254,6 +255,11 @@ private void testParse(String fileName) throws Exception {
         assertNull("testObject.myNullCollection", testObject.myNullCollection);
         assertEquals("testObject.myDefaultCollection", Collections.singleton("the one"),
                      testObject.myDefaultCollection);
+
+        Collection<String> expected = new ArrayList<>();
+        expected.add(null);
+        assertEquals(expected, testObject.myCollectionWithSingleNullValue);
```

Comment:
I would just do
```
assertEquals(CollectionUtils.newArrayList(null), testObject.myCollectionWithSingleNullValue);
```

Reply:
CollectionUtils would fail its preconditions check if I did that I believe.

Reply:
Oh, I forgot about that. Well, for future reference, you get around that by casting `null` to a `String`. 
```
CollectionUtils.newArrayList((String) null);
```
But you can leave it as is here.

Reply:
Woah, that's interesting. Thanks for the tip, and for the review!